pkg/clustermanager/applier.go

```diff
@@ -20,10 +20,12 @@ const (
 	applyClusterSpecTimeout           = 2 * time.Minute
 	waitForClusterReconcileTimeout    = time.Hour
 	retryBackOff                      = time.Second
-	waitForFailureMessageErrorTimeout = 20 * time.Second
+	waitForFailureMessageErrorTimeout = time.Minute
```

Comment:
I realized we would keep `defaultConditionCheckTotalCount` to at least one third of the wait time. Thus, I have changed `waitForFailureMessageErrorTimeout` to 1 minute.
The main reason is, we make count to 0 inside [waitFor](https://github.com/aws/eks-anywhere/pull/6781/files#diff-29080fc751d87a584f5bf80922c39b3467a34639557e8bdcf1e574a4ae719273R65) method if the condition doesn't match.
In case where the condition doesn't get match for some time and it matches at 15 seconds per say it needs 20 seconds more to check for the condition consistency.